react/Banner.tsx

```diff
@@ -70,7 +75,7 @@ const Banner = (props: Props) => {
       >
         <img
           className={classnames(handles.img, 'w-100 h-100')}
-          src={isMobile && mobileImage ? mobileImage : image}
+          src={device === "tablet" ? tabletImage : isMobile ? mobileImage : image}
```

Comment:
It should fallback to render `image` if `tabletImage` or `mobileImage` is empty.